roles/gitea/templates/app.ini.j2

```diff
@@ -118,8 +118,8 @@ FILE_EXTENSIONS = .md,.markdown,.mdown,.mkd
 [server]
 ; The protocol the server listens on. One of 'http', 'https', 'unix' or 'fcgi'.
 PROTOCOL = http
-DOMAIN = localhost
-ROOT_URL = %(PROTOCOL)s://%(DOMAIN)s:%(HTTP_PORT)s{{ gitea_url }}/
+DOMAIN = box
```

Comment:
@aidan-fitz I'm signing out to travel to LA now, but when I'm more awake later on / this wkd I'll catch up here.  My ill-informed question (apologies!) being...

Should the line `DOMAIN = box` be tied in with any Ansible variables like `iiab_hostname: box` at https://github.com/iiab/iiab/blob/master/vars/default_vars.yml#L59 ?

Or perhaps not!

Reply:
Generally, is it `{{ iiab_hostname }}.{{ iiab_domain }}` (currently `box.lan`)? Will both `box` and `box.lan` work?

Reply:
In my experience, `box.lan` has always been more reliable (owing to browser inconsistencies), but in theory `box` and `box.lan` are both supposed to work.